imprint/imprint.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous" />
+    <link rel="stylesheet" href="imprint.css" />
+
+    <title>Imprint</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-transparent d-flex">
+        <div class="container-fluid">
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link active" href="../index/index.html">Home</a>
+                    </li>
+                </ul>
+                <div class="my-2 my-lg-0 mr-sm-2">
+                    <button class="btn btn-outline-primary" onclick="location.href='../login/login_form.php'"
+                        type="button">Login</button>
+                    <button class="btn btn-primary"
+                        onclick="location.href='../register_personal_details/register_personal_details.html'"
+                        type="button">Register</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <main>
+        <div class="d-flex align-items-start justify-content-start">
+            <div>
+                <h1 class>Imprint</h1>
+                <h2>Information according to § 5 TMG:</h2>
+                <p><br>Oliver Dassinger</p>
+                <h3>Postal address:</h3>
+                <p>
+                <address>Industriestra&szlig;e 80a,
```

Comment:
Is the & and the ; necessary? Shouldn't utf-8 encoding do that automatically?

Reply:
There is no common consen about that